libs/components/datetime/src/lib/modules/date-pipe/date.service.ts

```diff
@@ -20,12 +31,26 @@ export class SkyDateFormatUtility {
   };
   /* spell-checker:enable */
 
-  public static format(
-    locale: string,
+  #defaultFormat = 'short';
+  #defaultLocale = 'en-US';
+  #ngUnsubscribe = new Subject<void>();
+
+  constructor() {
+    inject(SkyAppLocaleProvider)
+      .getLocaleInfo()
+      .pipe(takeUntil(this.#ngUnsubscribe))
```

Comment:
`ngUnsubscribe` is never being resolved. We need an `ngOnDestroy` here to do that.

Reply:
added